src/components/products/ProductItem.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from "react";
+import { Fragment } from "react";
+import axios from "axios";
+import Card from "../UI/Card";
+import './productItem.css'
+import Button from "../UI/Button";
+
+
+const ProductItem = ({ id, title, price, description, image }) => {
+
+
+let productId = id;
+
+console.log(productId)
+
+
+
+    const deleteProduct =(proId)=>{
+
+          // DELETE request using fetch with error handling
+          axios.get(`https://fakestoreapi.com/products/${proId}`, { method: 'DELETE' })
```

Comment:
You can simplify this by using `axios.delete(url)` instead of using two methods at once.